addons/logistics_wirecutter/script_component.hpp

```diff
@@ -88,7 +88,7 @@
 #define CUT_TIME_ENGINEER 7.5
 
 #define HAS_WIRECUTTER(unit) (\
-    "ACE_wirecutter" in (unit call EFUNC(common,uniqueItems)) \
+    count ((unit call EFUNC(common,uniqueItems)) select {_x isKindOf ["ACE_wirecutter", configFile >> "CfgWeapons"]}) > 0 \
```

Comment:
```suggestion
    (unit call EFUNC(common,uniqueItems)) findIf {_x isKindOf ["ACE_wirecutter", configFile >> "CfgWeapons"]} != -1 \
```

Will exit the loop when the first wirecutter is found